packit/upstream.py

```diff
@@ -514,7 +519,8 @@ def fix_spec(self, archive: str, version: str, commit: str):
         current_time = datetime.datetime.now().strftime("%Y%m%d%H%M%S%f")
         release = f"{original_release_number}.{current_time}{git_desc_suffix}"
 
-        msg = f"- Development snapshot ({commit})"
+        cmd = ["git", "log", "--pretty=format:- %s (%an)", "HEAD~%s..HEAD" % self._get_commit_count_since_last_tag()]
```

Comment:
`"HEAD~%s..HEAD" % self._get_commit_count_since_last_tag()`

There is no need to use the `<ref>~` syntax, you can just put the tag name in there.

Reply:
Hi @TomasTomecek. Thanks for the review. Sorry I do not understand what you mean.

Reply:
This
```python
"HEAD~%s..HEAD" % self._get_commit_count_since_last_tag()
```
should be equivalent to
```
last_tag = run_command(["git", "describe", "--tags", "--abbrev=0"], output=True).strip()
range = f"{last_tag}..HEAD"
```

And using tag explicitly instead of number of commits should be more stable.